### DISCUSSION TITLE
建议增加排序

### DISCUSSION THREAD

**Discussion by superen7527** [spencerwooo/onedrive-vercel-index#716] [Ideas]
按时间排序，这个挺重要的，希望可以加上。
还有界面中英文切换符号建议可以配置关闭，或者设置默认域名加载即中文而不是/zh_cn这样的链接

**Comment by myl7** [comment]
> 按时间排序

这个可以参见 #717

> 界面中英文切换符号建议……配置关闭……，设置默认域名加载即中文

默认语言可以通过修改 `next-i18next.config.js` 实现，具体参见 `next-i18next` 的文档，语言切换按钮不需要的话可以直接删除对应代码：

https://github.com/spencerwooo/onedrive-vercel-index/blob/7853fce88759d8876baf241a2f629f0a9f5acb92/components/Navbar.tsx#L92

**Reply by superen7527** [reply]
> > 按时间排序
> 
> 这个可以参见 #717
> 
> > 界面中英文切换符号建议……配置关闭……，设置默认域名加载即中文
> 
> 默认语言可以通过修改 `next-i18next.config.js` 实现，具体参见 `next-i18next` 的文档，语言切换按钮不需要的话可以直接删除对应代码：
> 
> https://github.com/spencerwooo/onedrive-vercel-index/blob/7853fce88759d8876baf241a2f629f0a9f5acb92/components/Navbar.tsx#L92

感谢，比如上传文件过多，没法按日期查找，搜索好像有时候也有问题。要是记不住名字文件又略多找起来很不方便，官方客户端虽然简陋但是也支持了这些排序。再者批量下载小文件也不好用，一直失败，所以才出此建议。修改语言我明天看看，再次感谢。

**Reply by myl7** [reply]
OneDrive 存在一部分功能是官方软件支持但 API 不提供的（除了排序，还有打包下载）。
这部分功能官方不愿意开放出来的话，开发者也没有（比较好的）解决方法。

**Reply by superen7527** [reply]
> OneDrive 存在一部分功能是官方软件支持但 API 不提供的（除了排序，还有打包下载）。 这部分功能官方不愿意开放出来的话，开发者也没有（比较好的）解决方法。

好的，我还以为搭成网址就是自己控制或者js，没想到还是和官方api有关系。辛苦了。
